pro/cypress/cypress.config.ts

```diff
@@ -1,11 +1,27 @@
 import { defineConfig } from 'cypress'
 import cypressFailFast = require('cypress-fail-fast/plugin')
+import fs from 'fs'
 
 // ts-unused-exports:disable-next-line
 export default defineConfig({
   e2e: {
     setupNodeEvents(on, config) {
       cypressFailFast(on, config)
+      on(
+        'after:spec',
+        (spec: Cypress.Spec, results: CypressCommandLine.RunResult) => {
+          if (results && results.video) {
+            // Do we have failures for any retry attempts?
+            const failures = results.tests.some((test) =>
+              test.attempts.some((attempt) => attempt.state === 'failed')
+            )
+            if (!failures) {
+              // delete the video if the spec passed and no tests retried
+              fs.unlinkSync(results.video)
+            }
+          }
+        }
+      )
```

Comment:
Plus besoin de ça, non ?

Reply:
si, quand les tests tournent, la vidéo est enregistrée. C'est seulement à la fin de l'exécution qu'on sait si on a eu un échec, et donc c'est seulement à ce moment-là qu'on peut supprimer la vidéo `after:spec`.
Au niveau du worflow on upload toutes les vidéos présentes...

Reply:
Ah, je n'avais pas compris ce point. Noté